packages/mattermost-redux/src/actions/users.ts

```diff
@@ -249,6 +249,7 @@ export function loadMe(): ActionFunc {
         const user = getState().entities.users.profiles[currentUserId];
         if (currentUserId) {
             Client4.setUserId(currentUserId);
+            await dispatch(getStatus(currentUserId));
```

Comment:
Do we need to wait for this to complete? Since we aren't able to fire off this request while we're loading everything else, perhaps we could just dispatch it without `awaiting` on it

Reply:
Yes, we do need to wait for this. This is because the `ResetStatusModal` component fetches the status only on the component mount (and not updated) and if the status is not found in the store, sets it to offline. Not awaiting here causes the status to always be displayed as offline.

Reply:
I don't think we can do that since it'll make the first load longer. I think we either need to load the status asynchronously while the other stuff loads (like in the `Promise.all` above) or we could make the modal handle the unloaded state properly